src/Log/Engine/BaseLog.php

```diff
@@ -95,12 +96,16 @@ protected function _format($data, array $context = [])
 
         $object = is_object($data);
```

Comment:
Nitpicking: I would call it ``$isObject`` as it increased readability of the conditions below.